### PR TITLE
Ft/manage team2

### PR DESCRIPTION
**Pull request by mami-now**
here is the footer of Team Managed

### CODE REVIEW

**Review by ishimweClaude12 — APPROVED** [review]
Great Job M.

**Review by Kingcedru — CHANGES_REQUESTED** [review]
Reduce the size of the Images and the texts

**Review by uwingabirejoselyne — APPROVED** [review]
good

File: src/input.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
```
Comment (uwingabirejoselyne) [inline src/input.css:3]: You don't  have to change  file of input.css .

**Review by Kingcedru — COMMENTED** [review]
Good

**Pull request merged**